tests/manage/mcg/test_bucket_policy.py

```diff
@@ -226,14 +242,39 @@ def test_object_actions(self, mcg_obj, bucket_factory):
             else:
                 raise UnexpectedBehaviour(f"{e.response} received invalid error code {response.error['Code']}")
 
```

Comment:
else:
                    assert False, "Get object succeeded when it should have failed"

Reply:
ack